ui/components/app/connected-sites-list/index.scss

```diff
@@ -13,6 +13,14 @@
     align-items: center;
     border-top: 1px solid #d2d8dd;
     padding: 16px 24px;
+
+    & &-link-button {
```

Comment:
The extra `&` resolves to the parent selector and is needed to elevate this selector's specificity above that of the `Button` class.